build/pipelines/templates/build-console-steps.yml

```diff
@@ -110,6 +110,7 @@ steps:
      $(Build.SourcesDirectory)/bin/$(RationalizedBuildPlatform)/$(BuildConfiguration)/*.dll
      $(Build.SourcesDirectory)/bin/$(RationalizedBuildPlatform)/$(BuildConfiguration)/*.xml
      **/Microsoft.VCLibs.*.appx
+     **/*unit.test*.dll

```

Comment:
Dustin and I discussed this: we weren't actually running any of 
```
UnitTests_TerminalCore\Terminal.Core.Unit.Tests.dll 
UnitTests_TerminalApp\Terminal.App.Unit.Tests.dll 
UnitTests_Remoting\Remoting.Unit.Tests.dll 
UnitTests_Control\Control.Unit.Tests.dll 
```

in CI, because they were never copied to the test output, because they were in subdirectories. Whoops!